nautobot/project-static/css/base.css

```diff
@@ -648,3 +648,8 @@ td span.hover_copy:hover .hover_copy_button {
 .theme-table .active-theme {
     border: 5px solid #FF0000;
 }
+
+.modal-body {
```

Comment:
Maybe add an explanatory comment (i.e. why this is needed, what issue it fixes)?